### PR TITLE
Fix chromium builds with is_cobalt = false

### PR DESCRIPTION
**Pull request by niranjanyardi**
b/374988048

Set is_cobalt = false for chromium only(non cobalt builds)

### CODE REVIEW

File: cobalt/build/configs/android-cobalt-arm/args.gn [hunk 1]
```diff
@@ -1,4 +1,2 @@
 target_os = "android"
 target_cpu = "arm"
-
-is_cobalt = true
```
Comment (andrewsavage1) [inline cobalt/build/configs/android-cobalt-arm/args.gn:4]: Add `is_cobalt = false` to the equivalent non-cobalt configs, i.e. https://github.com/youtube/cobalt/blob/main/cobalt/build/configs/android-arm/args.gn (there should be one for each of the cobalt configs)

Comment (niranjanyardi) [reply cobalt/build/configs/android-cobalt-arm/args.gn:4]: done

**Comment by niranjanyardi** [comment]
Blocked by https://github.com/youtube/cobalt/pull/4302

File: cobalt/build/configs/linux-x64/args.gn
Comment (andrewsavage1) [inline cobalt/build/configs/linux-x64/args.gn:1]: Can you check with the infra team if this file is actually ever used?

Comment (niranjanyardi) [reply cobalt/build/configs/linux-x64/args.gn:1]: pretty sure it is not used by this check: https://screenshot.googleplex.com/i7KcFadPzdsk9Tr
the gn.py will copy over the args.gn in this case.

see: https://github.com/youtube/cobalt/blob/main/cobalt/build/gn.py#L111

I'll remove it and see what happens

Comment (andrewsavage1) [inline cobalt/build/configs/linux-x64/args.gn:1]: @isarkis could you take a look at this and make sure it's fine to delete? Doesn't cause issues in the presubmit builds

**Review by andrewsavage1 — APPROVED** [review]

File: cobalt/build/configs/linux-x64/args.gn
Comment (niranjanyardi) [reply cobalt/build/configs/linux-x64/args.gn:1]: all the presubmits are passing (github actions) . did you mean some other presubmit ?

Comment (isarkis) [reply cobalt/build/configs/linux-x64/args.gn:1]: I am probably not the right person to answer that question.

**Pull request merged**